app/controllers/api/v1/base_controller.rb

```diff
@@ -1,31 +0,0 @@
-class Api::V1::BaseController < ActionController::Base
```

Comment:
Rubocop detected a number of offences, as it is not used at all, I decided to remove it. 
![screen shot 2017-03-27 at 09 19 57](https://cloud.githubusercontent.com/assets/16335156/24346966/98c2d668-12ce-11e7-91f0-fbf658387455.png)